wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -604,6 +604,35 @@ public void requireNotComposed(Collection<Command> commands) {
     }
   }
 
+  /**
+   * Requires that the specified command hasn't already been added to a composition, and is not
+   * currently scheduled.
+   *
+   * @param command The command to check
+   * @throws IllegalArgumentException if the given command has already been composed or scheduled.
+   */
+  public void requireNotComposedOrScheduled(Command command) {
```

Comment:
Is there a reason this is added as a separate function rather than modifying the existing?
On one hand, it's verbose and confusing, and the old method becomes rather redundant. On the other hand, this is more OCP-conformant.

Reply:
`RequireNotComposed` is still needed by `CommandScheduler`- we don't want to throw an exception if a command gets scheduled while it's running.